tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = foxpuppet
+
+[testenv:foxpuppet]
+deps =
+	pytest==3.0.0
```

Comment:
Latest is 3.0.3, also (just a nit) the indent seems excessive here - four spaces would be better.